packages/salesforcedx-vscode-core/src/commands/forceAuthDevHub.ts

```diff
@@ -22,6 +26,17 @@ import { ForceAuthDemoModeExecutor } from './forceAuthWebLogin';
 import { nls } from '../messages';
 import { isDemoMode } from '../modes/demo-mode';
 
+import * as vscode from 'vscode';
+
+import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import { isNullOrUndefined } from 'util';
```

Comment:
This method was deprecated, let's use our utility for this under `salesforcedx-utils-vscode`